setuptools/command/editable_wheel.py

```diff
@@ -335,7 +335,7 @@ def _safely_run(self, cmd_name: str):
             )
 
     def _create_wheel_file(self, bdist_wheel):
-        from wheel.wheelfile import WheelFile
+        from .._vendor.wheel.wheelfile import WheelFile
```

Comment:
I suppose this should be `extern` instead of `_vendor` right?

Reply:
Thank you very much @agronholm.

I suppose that the ones for typechecking that don't execute during runtime are fine (actually they might run better because typecheckers cannot handle MetaPathFinders). I think this one is the most important.